polkadot/runtime/kusama/src/lib.rs

```diff
@@ -761,6 +773,23 @@ impl pallet_treasury::Config for Runtime {
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
 	type SpendFunds = Bounties;
 	type SpendOrigin = TreasurySpender;
+	type AssetKind = VersionedLocatableAsset;
+	type Beneficiary = VersionedMultiLocation;
+	type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
+	type Paymaster = PayOverXcm<
+		TreasuryInteriorLocation,
+		crate::xcm_config::XcmRouter,
+		crate::XcmPallet,
+		ConstU32<{ 6 * HOURS }>,
+		Self::Beneficiary,
+		Self::AssetKind,
+		LocatableAssetConverter,
+		VersionedMultiLocationConverter,
```

Comment:
This needs a better trait bound. Currently it uses `TryConvert`, but it is entirely possible to reuse `WrapVersion`, which is exposed in `xcm::WrapVersion`, or `TryInto<xcm::latest::MultiLocation>`.

Reply:
With that, we can elimiante `VersionedMultiLocationConverter` -- that type is completely unnecessary, especially knowing that we already have `xcm::AlwaysLatest`.

Reply:
`AssetKind` might be just `u32` or anything else, same with `Beneficiary`, this is why I think it was set as `Convert` from the beginning.